### ISSUE TITLE
facebook social share -- proposed fix

### ISSUE THREAD

**Issue by richpauloo** [rstudio/flexdashboard#234]
The default social share for facebook currently doesn't work. 

To reproduce, visit any example flexdashboard online ([such as this one](https://beta.rstudioconnect.com/jjallaire/htmlwidgets-d3heatmap/htmlwidgets-d3heatmap.html)) and attempt to share via Facebook.

Digging into the HTML reveals that the request sent to facebook is:

```
https://www.facebook.com/sharer/sharer.php?s=100&p[url]="+encodeURIComponent(location.href)
```

Changing it to the following solves this issue:
```
https://www.facebook.com/sharer/sharer.php?u="+encodeURIComponent(location.href)
```

**Comment by gadenbuie** [comment]
Thanks! It's a bit hard to find in the Facebook developer docs, but [this page](https://developers.facebook.com/docs/plugins/share-button/) seems to agree with you!

**Comment by gadenbuie** [comment]
Fixed in e239f238294b093e9cd8c6e0920813717c1025b5, thanks for reporting this!
